docs/user_guide.md

```diff
@@ -335,14 +360,48 @@ module "forseti" {
       project_id         = "your-forseti-project-id-here"
       org_id             = "your-org-id-here"
       …
-      config_validator_enabled = true
+      config_validator_enabled      = true
```

Comment:
I think the TF module version also needs to be updated here `version = "~> 1.4"` *might* not work with the library sync if it's earlier than 4.1.0?

Reply:
Removed this reference, the sample provided in the docs will just use the latest version of the Forseti module.